doc/changelog/03-notations/19149-existing_level.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  :n:`at existing level`, this :n:`@syntax_modifier`
+  helps :ref:`factorize notations <NotationFactorization>`
+  with common prefixes
```

Comment:
```suggestion
  :n:`"at existing level"` option to :n:`@syntax_modifier`
  to help :ref:`factorize notations <NotationFactorization>`
  with common prefixes.  This is used in the :cmd:`Notation`, :cmd:`Reserved Infix'
  and :cmd:`Reserved Notation` commands
```

Reply:
This is technically true but the formers are only derivatives of the latter. Let's only point to `Reserved Notation` which seems to contain the most appropriate material in the refman in our case.